examples/all-clusters-app/esp32/main/DeviceCallbacks.cpp

```diff
@@ -30,6 +30,7 @@
 #include "WiFiWidget.h"
 #include "esp_heap_caps.h"
 #include "esp_log.h"
+#include <Bluetooth.h>
```

Comment:
This needs better renaming

Reply:
Is it okay to rename it as `BLEdeinit.h` ?